File: Programmers/Inho/Level01/문자열 다루기 기본.cs
using System;
public class Solution
{
    public bool solution(string s)
    {
        bool answer = true;
        if (s.Length != 4 && s.Length != 6)
            return false;
        foreach (char a in s)
        {
            if (!(a >= '0' && a <= '9'))
                return false;
        }
        return answer;
    }
}

File: Programmers/Inho/C++/Level01/문자열 다루기 기본.cpp
#include <string>
#include <vector>

using namespace std;

bool solution(string s) {
    bool answer = true;
    for(int i = 0;i<s.size();i++){
        if(s[i]<'0'||s[i]>'9')
            return false;
    }
    return answer;
}